packages/now-routing-utils/src/index.ts

```diff
@@ -47,9 +54,23 @@ export function normalizeRoutes(inputRoutes: Route[] | null): NormalizedRoutes {
 
   for (const route of routes) {
     if (isHandler(route)) {
-      if (Object.keys(route).length !== 1) {
+      if (route.handle === 'error') {
+        const invalidKeys = Object.keys(route).filter(key => {
+          return !VALID_HANDLE_ERROR_KEYS.includes(key);
```

Comment:
```suggestion
          return !validHandleErrorKeys.has(key);
```

Then you can use Set.has()